Makefile

```diff
@@ -147,7 +147,6 @@ protos := \
 	proto/spire/common/common.proto \
 
 api-protos := \
```

Comment:
api-protos will be empty we want to keep this var?

Reply:
I stopped short of removing all of the infrastructure for generating API protos from the SPIRE Makefile. We haven't quite figured out how we want to stage new APIs and it's possible they live in SPIRE until they are ready to "graduate" and move to the API SDK.